gloo/docs/operator_guide/usage_statistics.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Usage Statistics
+weight: 3
+---
+
+## Usage Statistics
+
+Gloo will periodically collect usage information from running instances. The details of this collection can be found [here](https://github.com/solo-io/go-checkpoint).
```

Comment:
When does it get checked? I think on Gloo startup and also Glooctl invocation?

Reply:
Basically one happens on startup, and then on a given interval after that

Reply:
👍